src/calculateRentalCost.js

```diff
@@ -3,8 +3,26 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const totalRentPrice = days * rentCost;
+  const longTermRent = 7;
+  const shortTermRent = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days < shortTermRent) {
+    return totalRentPrice;
+  }
+
+  if (days >= shortTermRent && days <= 6) {
+    return totalRentPrice - shortTermDiscount;
```

Comment:
The condition `days <= 6` is unnecessary because if `days` is 7 or more, the next `if` condition will handle it. You can remove this check to simplify the logic, adhering to the checklist item 5.